sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c

```diff
@@ -707,7 +707,7 @@ bool gnrc_ndp_internal_pi_opt_handle(kernel_pid_t iface, uint8_t icmpv6_type,
     }
 #endif
     prefix = gnrc_ipv6_netif_find_addr(iface, &pi_opt->prefix);
-    if (((prefix == NULL) ||
+    if ((pi_opt->flags & NDP_OPT_PI_FLAGS_A) && ((prefix == NULL) ||
```

Comment:
At least the prefix should be added, when the auto-configuration is not set, otherwise, why send a PIO at all?